adafruit_ra8875/ra8875.py

```diff
@@ -0,0 +1,617 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2019 Melissa LeBlanc-Williams for Adafruit Industries
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+"""
+`adafruit_ra8875.ra8875`
+====================================================
+
+A Driver Library for the RA8875
+
+* Author(s): Melissa LeBlanc-Williams
+
+Implementation Notes
+--------------------
+
+**Hardware:**
+
+* RA8875 Driver Board for 40-pin TFT Touch Displays - 800x480:
+  https://www.adafruit.com/product/1590
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+* Adafruit's Bus Device library:
+  https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
+"""
+
+# imports
+import time
+from digitalio import Direction
+import adafruit_bus_device.spi_device as spi_device
+import adafruit_ra8875.registers as reg
+
+try:
+    import struct
+except ImportError:
+    import ustruct as struct
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_RA8875.git"
+
+#pylint: disable-msg=invalid-name
+def color565(r, g=0, b=0):
+    """Convert red, green and blue values (0-255) into a 16-bit 565 encoding."""
+    try:
+        r, g, b = r  # see if the first var is a tuple/list
+    except TypeError:
+        pass
+    return (r & 0xf8) << 8 | (g & 0xfc) << 3 | b >> 3
+#pylint: enable-msg=invalid-name
+
+class RA8875_Device(object):
+    """Set Initial Variables"""
```

Comment:
Can you add parameter descriptions for the class, so they're available in the documentation? 
Example:
```
"""
Writes the specified 'payload' to the sensor
Returns the results of the write attempt.

:param bytearray payload: The byte array to write to the sensor
:param rxlen: (optional) The numbers of bytes to read back (default=0)
"""
```
Requesting this for each class, as well as each public function (private function docs aren't built by default, though can be forced).

Reply:
Ok, I added parameters for all functions. I used http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists as a reference.